### DISCUSSION TITLE
NPM Workspaces Support in CI

### DISCUSSION THREAD

**Discussion by artmsilva** [codesandbox/codesandbox-client#6623] [Q&A]
Hi, I recently started to set up CI in this PR https://github.com/washingtonpost/wpds-ui-kit/pull/91 however I hit a wall trying to get CI to use NPM v8. Our org has been using NPM 8.3.0 for a while now. It looks like CI still doesn't support NPM 7 or 8 (#5249). Any timeline or roadmap on adding support?

**Comment by JamesACS** [comment]
Hey @artmsilva - This is on our to-do list but I'm afraid I don't have a timeframe just yet!
